modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -226,6 +234,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     if (this._source === undefined) {
       throw new BuildTransactionError('Invalid transaction: missing source');
     }
+    this.validateAddress(this._source);
```

Comment:
Please also change the definition for `validateAddress()` to use `isValidAddress` from './utils' instead